### PR TITLE
Update benchmarks for ODL 8

### PR DESCRIPTION
**Pull request by habbes**
Update ODL benchmarks for ODL 8 and .NET 8.

### CODE REVIEW

File: test/PerformanceTests/SerializationComparisonsTests/Lib/WriterHelpers.cs [hunk 1]
```diff
@@ -19,25 +20,25 @@ public static class WriterHelpers
...
+                stream, false, encoding ?? Encoding.UTF8);
         }
 
-        public static IJsonWriterAsync CreateODataJsonWriterAsync(this Stream stream, Encoding encoding = null)
+        public static IJsonWriter CreateODataJsonWriterAsync(this Stream stream, Encoding encoding = null)
         {
-            return jsonWriterFactory.CreateAsynchronousJsonWriter(
-                new StreamWriter(stream, encoding ?? Encoding.UTF8), false);
+            return jsonWriterFactory.CreateJsonWriter(
+                stream, false, encoding ?? Encoding.UTF8);
```
Comment (ElizabethOkerio) [inline test/PerformanceTests/SerializationComparisonsTests/Lib/WriterHelpers.cs:29]: Can we use a named parameter here to make it easy for someone to understand what the `false` for example stands for?

Comment (ElizabethOkerio) [reply test/PerformanceTests/SerializationComparisonsTests/Lib/WriterHelpers.cs:29]: Update this in all other places. otherwise the pr looks good.

Comment (habbes) [reply test/PerformanceTests/SerializationComparisonsTests/Lib/WriterHelpers.cs]: Updated.

**Comment by habbes** [comment]
This has already been merged.

**Pull request closed without merging**